recipe/meta.yaml

```diff
@@ -33,3 +33,5 @@ about:
 extra:
   recipe-maintainers:
     - arokem
+    - glemaitre
+    - darcymason
```

Comment:
Just to confirm, are you ok being added as a maintainer, @darcymason?

Reply:
Yes